src/exts/js/testing.ts

```diff
@@ -351,7 +339,7 @@ function test(description: string, fn: () => void) {
   } catch (error: any) {
     if (error instanceof AssertionError) {
       console.log(
-        `❌ ${description}\n   ${error.message}\n   ${error.getDiff(3)}`,
+        `❌ ${description}\n   ${error.message}\n   ${error.getDiff(3)}`
```

Comment:
@vr-varad do you think we could make this more verbose?

Reply:
yeah sure!!